src/demo/feature-toggle/feature-toggle.example.component.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChangeDetectorRef,
+  Component,
+  OnInit, ViewChild
+} from '@angular/core';
+
+import {FeatureTogglesService} from '../../app/service/feature-toggles.service';
+import {FeatureToggleComponent} from '../../app/feature-wrapper/feature-toggle.component';
+
+@Component({
+  selector: 'feature-toggle-example',
+  styles: [`
+    .sample-form .form-horizontal .form-group {
+      margin-left: 0px;
+    }
+    .padding-top-15 {
+      padding-top: 15px;
+    }
+    .padding-bottom-15 {
+      padding-bottom: 15px;
+    }
+  `],
+  templateUrl: './feature-toggle.example.component.html'
+})
+export class FeatureToggleExampleComponent implements OnInit {
+  // we need a reference to refresh the component depending on mock values, on this demo page.
+  // not needed usually
+  @ViewChild(FeatureToggleComponent) toggleComponent: FeatureToggleComponent;
+  featureFlagName: string = 'Test';
+  featureFlagEnable: boolean = true;
+  featureFlagEnablementLevel: string = 'beta';
+  userLevel: string = 'internal';
+
+  constructor(private featureToggleService: FeatureTogglesService, private cd: ChangeDetectorRef) {
+  }
+
+  ngOnInit(): void {
+    (this.featureToggleService as any).featureFlagName = this.featureFlagName;
```

Comment:
Again with the type casting.

Reply:
Casting here is bc FeatureToggleServiceMock is injected for Type FeatureToggleService but only the Mock contains those inputs. Bear in mind this is the demo part.